test-app/bundles/educandu-page-template.less

```diff
@@ -0,0 +1,126 @@
+@educandu-page-template-header-content-min-height: 32px;
+@educandu-page-template-header-z-index: 1;
+@educandu-page-template-content-z-index: 0;
+@educandu-page-template-footer-z-index: 0;
+
+.EducanduPageTemplate {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  min-height: 100vh;
+  position: relative;
+  flex-direction: column;
+  min-width: @global-min-page-width;
+  max-width: 100vw;
+}
+
+.EducanduPageTemplate-headerArea {
+  top: 0;
+  flex: none;
+  width: 100%;
+  z-index: 1000;
+  position: sticky;
+  z-index: @educandu-page-template-header-z-index;
+  margin-bottom: @global-content-padding;
+}
+
+.EducanduPageTemplate-header {
+  display: flex;
+  align-items: center;
+  min-height: @educandu-page-template-header-content-min-height + (2* @global-content-padding);
```

Comment:
Small thing: I would also extract this to the top in a variable, I feel it's easier to adjust/"debug" when you see all involved values defined right there

Reply:
Sure, done.